ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRow.java

```diff
@@ -55,7 +59,8 @@
  * <p>We also perform some manipulation of the parsed JSON to match details of our table
  * schemas in BigQuery.
  */
-public class PubsubMessageToTableRow {
+public class PubsubMessageToTableRow
+    extends MapElementsWithErrors<PubsubMessage, KV<TableDestination, TableRow>> {
```

Comment:
If we mark this class as `implements Serializable`, that should be sufficient to get the tests passing again. `PTransform` implements Serializable, so removing the subclass here also removed the serializable interface.

Reply:
Okay, I see. I created a follow-up in https://github.com/mozilla/gcp-ingestion/pull/1143